CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 All notable changes for each version of this project will be documented in this file.
 
+# 7.2.5
+- `IgxList`
+    - **Breaking Change** the `index` property is now an `@Input` and can be assigned by structural directives such as `*igxFor`.
+    ```html
+        <igx-list>
+            <div [style.height]="'480px'" [style.overflow]="'hidden'" [style.position]="'relative'">
+                <igx-list-item [index]="i" *igxFor="let item of data; index as i; scrollOrientation: 'vertical'; containerSize: '480px'; itemSize: '48px'">
+                    <div>{{ item.key }}</div>
+                    <div class="contact__info">
+                        <span class="name">{{item.name}}</span>
+                    </div>
+                </igx-list-item>
+            </div>
+        </igx-list>
+    ```
+    - The `items` property now returns the items sorted by their index.
```

Comment:
This was/is also true when using non-virtualized items until now, right?
So the note should probably be something like: "When using a virtualized `igxFor` the `items`  property returns the items sorted by their index, while `children` order is not guaranteed"